docs/troubleshooting.md

```diff
@@ -109,20 +118,26 @@ If everything seems healthy, and still no packages show up in the UI, it could b
 
 ## OperatorSource failing to download from datastore
 
-If an OperatorSource fails to reconcile for any reason (i.e operators in that particular OperatorSource fail to show up in the UI), it might be helpful first inspect the `STATUS` and the `MESSAGE` of the OperatorSource with `oc get opsrc -n openshift-marketplace`. You can also inspect the log messages of the marketplace-operator for detailed information about the OperatorSource. 
+If an OperatorSource fails to reconcile for any reason, first inspect the `STATUS` and the `MESSAGE` of the OperatorSource with `oc get opsrc -n openshift-marketplace`. You can also inspect the log messages of the marketplace-operator with `oc logs <name-of-marketplace-operator> -n openshift-marketplace` for detailed information about the OperatorSource. 
```

Comment:
It will help to call out that you will find connection errors here. It is too open ended as it stands now.

Reply:
@aravindhp merged `OperatorSource failing to download` and `OperatorSource failing to connect to datastore` into one section.